.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+blank_issues_enabled: true
+contact_links:
+  - name: "Questions, Help and Support"
+    url: https://github.com/scikit-image/scikit-image/blob/main/.github/SUPPORT.md
+    about: >
+      If you have a question, please look at the listed resources available on
+      our support page.
+  - name: "StackOverflow"
```

Comment:
If we explicitly link to StackOverflow, why don't we also expand the SUPPORT.md link above and include those here? I think that's a good way to go, to mimic https://scikit-image.org